image.yaml

```diff
@@ -1,25 +1,25 @@
 schema_version: 1
 
 name: "redhat-sso-7-tech-preview/sso-cd"
-description: "Red Hat Single Sign-On 7.4 continuous delivery container image"
-version: "7.4.0.CD05"
+description: "Red Hat Single Sign-On continuous delivery container image"
```

Comment:
Same here? IOW replace with `Red Hat Single Sign-On Continuous Delivery (RH-SSO CD) for OpenShift`

Reply:
I changed this to `Continuous Delivery`